src/locales/en.json

```diff
@@ -143,7 +143,12 @@
     },
     "prevalence_info_source_information": "Prevalence data consolidated from <1>Johns Hopkins CSSE</1> (reported cases), <4>Covid Act Now</4> (US positive test rates), <7>Our World in Data</7> (international positive test rates), and <10>Coronavirus COVID-19 România</10> (Romania reported cases).",
     "select_scenario": "Start with a premade scenario or <2>build your own</2>:",
-    "no_prebuilt_scenario_found": "We don't have this scenario yet, but you can build your own below.",
+    "no_prebuilt_scenario_found": "No premade scenario found, but you may",
```

Comment:
These two need to be a single entry; other languages may not have the same clause order.

Reply:
removed the "but you may".